Tests are failing on main

Seems conda-related: https://github.com/microsoft/vscode-python/runs/4900770674?check_suite_focus=true

Started failing with c63d38d148aec3a7b7a3890c7b3d9bfefbca36f5

```
1) PythonExecutableService
       Importing without a valid PYTHONPATH should fail:
     Error: No matching bindings found for serviceIdentifier: Symbol(IConfigurationService)
      at _validateActiveBindingCount (node_modules/inversify/lib/planning/planner.js:63:23)
      at _getActiveBindings (node_modules/inversify/lib/planning/planner.js:49:5)
      at _createSubRequests (node_modules/inversify/lib/planning/planner.js:86:26)
      at Object.plan (node_modules/inversify/lib/planning/planner.js:137:9)
      at /home/runner/work/vscode-python/vscode-python/path with spaces/node_modules/inversify/lib/container/container.js:319:37
      at Container._get (node_modules/inversify/lib/container/container.js:312:44)
      at Container.get (node_modules/inversify/lib/container/container.js:232:21)
      at ServiceContainer.get (out/client/ioc/container.js:24:89)
      at Object.getPythonSetting (out/client/pythonEnvironments/common/externalDependencies.js:112:47)
      at getCandidates (out/client/pythonEnvironments/common/environmentManagers/conda.js:111:60)
      at getCandidates.next (<anonymous>)
      at Function.locate (out/client/pythonEnvironments/common/environmentManagers/conda.js:177:26)
      at Function.getConda (out/client/pythonEnvironments/common/environmentManagers/conda.js:102:39)
      at CondaService.getCondaVersion (out/client/pythonEnvironments/common/environmentManagers/condaService.js:44:30)
      at PythonExecutionFactory.createCondaExecutionService (out/client/common/process/pythonExecutionFactory.js:96:31)
      at PythonExecutionFactory.create (out/client/common/process/pythonExecutionFactory.js:61:50)
      at Context.<anonymous> (out/test/common/process/pythonProc.simple.multiroot.test.js:50:35)


  2) PythonExecutableService
       Importing with a valid PYTHONPATH from .env file should succeed:
     Error: No matching bindings found for serviceIdentifier: Symbol(IConfigurationService)
      at _validateActiveBindingCount (node_modules/inversify/lib/planning/planner.js:63:23)
      at _getActiveBindings (node_modules/inversify/lib/planning/planner.js:49:5)
      at _createSubRequests (node_modules/inversify/lib/planning/planner.js:86:26)
      at Object.plan (node_modules/inversify/lib/planning/planner.js:137:9)
      at /home/runner/work/vscode-python/vscode-python/path with spaces/node_modules/inversify/lib/container/container.js:319:37
      at Container._get (node_modules/inversify/lib/container/container.js:312:44)
      at Container.get (node_modules/inversify/lib/container/container.js:232:21)
      at ServiceContainer.get (out/client/ioc/container.js:24:89)
      at Object.getPythonSetting (out/client/pythonEnvironments/common/externalDependencies.js:112:47)
      at getCandidates (out/client/pythonEnvironments/common/environmentManagers/conda.js:111:60)
      at getCandidates.next (<anonymous>)
      at Function.locate (out/client/pythonEnvironments/common/environmentManagers/conda.js:177:26)
      at Function.getConda (out/client/pythonEnvironments/common/environmentManagers/conda.js:102:39)
      at CondaService.getCondaVersion (out/client/pythonEnvironments/common/environmentManagers/condaService.js:44:30)
      at PythonExecutionFactory.createCondaExecutionService (out/client/common/process/pythonExecutionFactory.js:96:31)
      at PythonExecutionFactory.create (out/client/common/process/pythonExecutionFactory.js:61:50)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at Context.<anonymous> (out/test/common/process/pythonProc.simple.multiroot.test.js:62:35)

  3) PythonExecutableService
       Known modules such as 'os' and 'sys' should be deemed 'installed':
     Error: No matching bindings found for serviceIdentifier: Symbol(IConfigurationService)
      at _validateActiveBindingCount (node_modules/inversify/lib/planning/planner.js:63:23)
      at _getActiveBindings (node_modules/inversify/lib/planning/planner.js:49:5)
      at _createSubRequests (node_modules/inversify/lib/planning/planner.js:86:26)
      at Object.plan (node_modules/inversify/lib/planning/planner.js:137:9)
      at /home/runner/work/vscode-python/vscode-python/path with spaces/node_modules/inversify/lib/container/container.js:319:37
      at Container._get (node_modules/inversify/lib/container/container.js:312:44)
      at Container.get (node_modules/inversify/lib/container/container.js:232:21)
      at ServiceContainer.get (out/client/ioc/container.js:24:89)
      at Object.getPythonSetting (out/client/pythonEnvironments/common/externalDependencies.js:112:47)
      at getCandidates (out/client/pythonEnvironments/common/environmentManagers/conda.js:111:60)
      at getCandidates.next (<anonymous>)
      at Function.locate (out/client/pythonEnvironments/common/environmentManagers/conda.js:177:26)
      at Function.getConda (out/client/pythonEnvironments/common/environmentManagers/conda.js:102:39)
      at CondaService.getCondaVersion (out/client/pythonEnvironments/common/environmentManagers/condaService.js:44:30)
      at PythonExecutionFactory.createCondaExecutionService (out/client/common/process/pythonExecutionFactory.js:96:31)
      at PythonExecutionFactory.create (out/client/common/process/pythonExecutionFactory.js:61:50)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at Context.<anonymous> (out/test/common/process/pythonProc.simple.multiroot.test.js:70:35)

  4) PythonExecutableService
       Unknown modules such as 'xyzabc123' be deemed 'not installed':
     Error: No matching bindings found for serviceIdentifier: Symbol(IConfigurationService)
      at _validateActiveBindingCount (node_modules/inversify/lib/planning/planner.js:63:23)
      at _getActiveBindings (node_modules/inversify/lib/planning/planner.js:49:5)
      at _createSubRequests (node_modules/inversify/lib/planning/planner.js:86:26)
      at Object.plan (node_modules/inversify/lib/planning/planner.js:137:9)
      at /home/runner/work/vscode-python/vscode-python/path with spaces/node_modules/inversify/lib/container/container.js:319:37
      at Container._get (node_modules/inversify/lib/container/container.js:312:44)
      at Container.get (node_modules/inversify/lib/container/container.js:232:21)
      at ServiceContainer.get (out/client/ioc/container.js:24:89)
      at Object.getPythonSetting (out/client/pythonEnvironments/common/externalDependencies.js:112:47)
      at getCandidates (out/client/pythonEnvironments/common/environmentManagers/conda.js:111:60)
      at getCandidates.next (<anonymous>)
      at Function.locate (out/client/pythonEnvironments/common/environmentManagers/conda.js:177:26)
      at Function.getConda (out/client/pythonEnvironments/common/environmentManagers/conda.js:102:39)
      at CondaService.getCondaVersion (out/client/pythonEnvironments/common/environmentManagers/condaService.js:44:30)
      at PythonExecutionFactory.createCondaExecutionService (out/client/common/process/pythonExecutionFactory.js:96:31)
      at PythonExecutionFactory.create (out/client/common/process/pythonExecutionFactory.js:61:50)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at Context.<anonymous> (out/test/common/process/pythonProc.simple.multiroot.test.js:77:35)

  5) PythonExecutableService
       Ensure correct path to executable is returned:
     Error: No matching bindings found for serviceIdentifier: Symbol(IConfigurationService)
      at _validateActiveBindingCount (node_modules/inversify/lib/planning/planner.js:63:23)
      at _getActiveBindings (node_modules/inversify/lib/planning/planner.js:49:5)
      at _createSubRequests (node_modules/inversify/lib/planning/planner.js:86:26)
      at Object.plan (node_modules/inversify/lib/planning/planner.js:137:9)
      at /home/runner/work/vscode-python/vscode-python/path with spaces/node_modules/inversify/lib/container/container.js:319:37
      at Container._get (node_modules/inversify/lib/container/container.js:312:44)
      at Container.get (node_modules/inversify/lib/container/container.js:232:21)
      at ServiceContainer.get (out/client/ioc/container.js:24:89)
      at Object.getPythonSetting (out/client/pythonEnvironments/common/externalDependencies.js:112:47)
      at getCandidates (out/client/pythonEnvironments/common/environmentManagers/conda.js:111:60)
      at getCandidates.next (<anonymous>)
      at Function.locate (out/client/pythonEnvironments/common/environmentManagers/conda.js:177:26)
      at Function.getConda (out/client/pythonEnvironments/common/environmentManagers/conda.js:102:39)
      at CondaService.getCondaVersion (out/client/pythonEnvironments/common/environmentManagers/condaService.js:44:30)
      at PythonExecutionFactory.createCondaExecutionService (out/client/common/process/pythonExecutionFactory.js:96:31)
      at PythonExecutionFactory.create (out/client/common/process/pythonExecutionFactory.js:61:50)
      at Context.<anonymous> (out/test/common/process/pythonProc.simple.multiroot.test.js:95:35)
```

```
  1) PythonExecutableService
       Importing with a valid PYTHONPATH from .env file should succeed:

      AssertionError: expected { Object (stdout, stderr) } to have property 'stdout' of 'Hello\r\n', but got 'Hello\n'
      + expected - actual

      -Hello
      +Hello
      
      at d:\a\vscode-python\vscode-python\path with spaces\node_modules\chai-as-promised\lib\chai-as-promised.js:302:22
      at runMicrotasks (<anonymous>)
      at processTicksAndRejections (internal/process/task_queues.js:93:5)
      at Context.<anonymous> (d:\a\vscode-python\vscode-python\path with spaces\out\test\common\process\pythonProc.simple.multiroot.test.js:65:9)
```

Failing CI due to an extra `\r` character: https://github.com/microsoft/vscode-python/runs/4986976848?check_suite_focus=true

--- a/src/test/common/process/pythonProc.simple.multiroot.test.ts
+++ b/src/test/common/process/pythonProc.simple.multiroot.test.ts
@@ -7,7 +7,6 @@ import { expect, use } from 'chai';
 import * as chaiAsPromised from 'chai-as-promised';
 import { execFile } from 'child_process';
 import * as fs from 'fs-extra';
-import { EOL } from 'os';
 import * as path from 'path';
 import { ConfigurationTarget, Uri } from 'vscode';
 import { IPythonExecutionFactory, StdErrError } from '../../../client/common/process/types';
@@ -75,12 +74,12 @@ suite('PythonExecutableService', () => {
     test('Importing with a valid PYTHONPATH from .env file should succeed', async () => {
         await configService.updateSetting('envFile', undefined, workspace4PyFile, ConfigurationTarget.WorkspaceFolder);
         const pythonExecService = await pythonExecFactory.create({ resource: workspace4PyFile });
-        const promise = pythonExecService.exec([workspace4PyFile.fsPath], {
+        const result = await pythonExecService.exec([workspace4PyFile.fsPath], {
             cwd: path.dirname(workspace4PyFile.fsPath),
             throwOnStdErr: true,
         });
 
-        await expect(promise).to.eventually.have.property('stdout', `Hello${EOL}`);
+        expect(result.stdout.startsWith('Hello')).to.be.equals(true);
     }).timeout(TEST_TIMEOUT * 3);
 
     test("Known modules such as 'os' and 'sys' should be deemed 'installed'", async () => {

